osu.Game.Rulesets.Taiko/UI/TaikoPlayfieldAdjustmentContainer.cs

```diff
@@ -24,10 +25,17 @@ protected override void Update()
             // Players coming from stable expect to be able to change the aspect ratio regardless of the window size.
             // We originally wanted to limit this more, but there was considerable pushback from the community.
             //
-            // As a middle-ground, the aspect ratio can still be adjusted in the downwards direction but has a maximum limit.
+            // As a middle-ground, the aspect ratio can still be adjusted in the downwards direction but has a maximum limit (5:4 aspect).
             // This is still a bit weird, because readability changes with window size, but it is what it is.
-            if (LockPlayfieldMaxAspect.Value && Parent.ChildSize.X / Parent.ChildSize.Y > default_aspect)
-                height *= Math.Clamp(Parent.ChildSize.X / Parent.ChildSize.Y, 0.4f, 4) / default_aspect;
```

Comment:
Removing this clamp breaks things on wide aspect ratios, the playfield seemingly gets unloaded?

https://user-images.githubusercontent.com/54949148/218187388-a164064b-2388-4d60-b820-0179f2e6eef4.mp4

Reply:
that looks cropped out by masking moreso than unloaded

Reply:
Correct, this is a masking/presence concern. Doesn't change the fact that it probably needs to be addressed somehow.